src/status_im2/contexts/communities/events.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im2.contexts.communities.events
+  (:require [utils.re-frame :as rf]))
+
+(rf/defn enable-community-channels-mocks
+  {:events [:mocks/toggle-community-channels-mocking]}
```

Comment:
im not sure , why we need this ? we should mock only in test, no code for mocks should be in app code